### PR TITLE
Basic part translation into Korean

### PR DESCRIPTION
**Pull request by SaidBySolo**
image

### CODE REVIEW

**Comment by ahopkins** [comment]
Awesome. Let @sanic-org/docs know if you have any questions or run into any problems.

**Comment by ZinkLu** [comment]
Great! Let us know if you have any issue in doc content or vuepress.

File: src/ko/guide/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+저희의 스폰서에게 감사합니다. [오픈 콜렉티브](https://opencollective.com/sanic-org)를 확인하여 Sanic 자금 지원에 대해 자세히 알아보십시오.
+
+## 커뮤니티에 참가
+
+토론을위한 메인 채널은 [커뮤니티 포럼](https://community.sanicframework.org/)입니다. 또한 [디스코드 서버](https://discord.gg/FARQzAEMAA)에서 실시간 토론및 채팅이 가능합니다.
+
+Stackoverflow의 `[sanic]` 태그는 프로젝트 유지보수자들에 의해 [적극적으로 모니터링](https://stackoverflow.com/questions/tagged/sanic)되고 있습니다.
+
+## Contribution
```
Comment (wonderlandpark) [inline src/ko/guide/README.md:59]:
```suggestion
## 기여
```

File: src/ko/guide/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+|         |                                                                                                                         |
+| ------- | ----------------------------------------------------------------------------------------------------------------------- |
+| Build   | [![Build Status][]][1] [![AppVeyor Build Status][]][2] [![Codecov]][3]                                                  |
+| Docs    | [![Documentation]][4]                                                                                                   |
+| Package | [![PyPI][]][5] [![PyPI version][]][5] [![PyPI Wheel][]][6] [![Supported implementations][]][6] [![Code style black]][7] |
+| Support | [![Forums][]][8] [![Discord][]][9] [![Awesome Sanic List]][10]                                                          |
+| Stats   | [![Downloads][]][11] [![Downloads][12]][11]                                                                             |
+
+## 이게 뭐야?
```
Comment (wonderlandpark) [inline src/ko/guide/README.md:17]: How about this
```suggestion
## 이게 뭔가요?
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

**Comment by miss85246** [comment]
Haha, we have to support one more language, let sanic guide become more powerful 🎸

**Comment by ahopkins** [comment]
Waiting for @vltr to translate it to Portuguese.

**Comment by ahopkins** [comment]
@SaidBySolo How's it coming? Do you need anything from us non-Korean speakers?

**Comment by SaidBySolo** [comment]
> @SaidBySolo 어떻게 되나요? 한국어가 아닌 우리에게 필요한 것이 있습니까?

Progress is slowing due to academic problems. :(

i am aim to work so that we can merge by the end of next month!

**Comment by miss85246** [comment]
Come on！ we support you in spirit
힘내, 우리는 정신적으로 너를 지지한다
@SaidBySolo

**Comment by SaidBySolo** [comment]
@sanic-org/docs

Is it okay to divide the pull request by part?
If I am lucky, I will finish the Basic part today.

**Comment by ahopkins** [comment]
Yeah. That works for me.

**Comment by ahopkins** [comment]
Where do we stand with this?

**Comment by SaidBySolo** [comment]
> Where do we stand with this?


It seems that all the basic parts of version 21.3 have been completed in the local commit.
I will open a PR tomorrow after merging upstream and making corrections.

**Comment by SaidBySolo** [comment]
Okay, I have time, so I finished today. 
If I have time, I will definitely work on the rest.

Based on this PR, I hope that other Koreans will also be interested in sanic.

ps. Sorry for taking so long. I forgot this PR. :(

**Review by ahopkins — APPROVED** [review]

**Comment by ahopkins** [comment]
:clap: :champagne: 
Thanks for all the work on this!

**Pull request merged**